src/api/pastel-rpc/__tests__/transactions.test.ts

```diff
@@ -75,13 +75,23 @@ describe('api/pastel-rpc/transactions', () => {
   })
 
   test('can’t get the transaction', async () => {
+    expect.hasAssertions()
+
     function callback(alltxlist: TListTransactions[]): void {
       expect(alltxlist).toBeNaN()
     }
+
+    const mock = new MockAdapter(axios)
+    const data = undefined
+
+    mock.onPost(config.url).reply(500, data)
+
     try {
       await fetchTandZTransactions(config, callback)
     } catch (err) {
-      expect(err.message).not.toBeNull()
+      expect(err.message).toEqual(
+        'api/pastel-rpc server error: [object Object]',
```

Comment:
Hm, seems like there is a bug in the `rpc` function. Could you fix it, so it gives the proper error message and not `[object Object]`? Maybe a `JSON.stringify()` would help

Reply:
it has been updated.